static/app/views/performance/browser/webVitals/pageOverview.tsx

```diff
@@ -216,6 +229,22 @@ export default function PageOverview() {
                   <DatePageFilter />
                 </PageFilterBar>
               </TopMenuContainer>
+              {shouldUseStoredScores && !isDismissed && (
+                <StyledAlert type="info" showIcon>
+                  <AlertContent>
+                    {t(
+                      'We changed how Performance Scores are calculated for your projects.'
```

Comment:
I find this message a little uninformative. I feel like maybe once we get the blog up we can link it here, but in the meantime could we mention the date here so it contextualizes the line too?

Something like "We changed how Performance Scores are calculated for your projects. After [X date], you are getting the latest and greatest" but better.

Reply:
👍 gunna update this in a follow up pr